src/Tribe/Main.php

```diff
@@ -605,6 +606,40 @@ public function add_help_section_support_content( $help ) {
 		}
 
 		/**
+		 * Append the text about The Events Calendar to the Extra Help section on the Help page
+		 *
+		 * @filter "tribe_help_pre_get_sections"
+		 * @param Tribe__Admin__Help_Page $help The Help Page Instance
+		 * @return void
+		 */
+		public function add_help_section_extra_content( $help ) {
+			if ( ! $help->is_active( array( 'events-calendar-pro', 'event-tickets-plus' ) ) && $help->is_active( 'event-tickets' ) ) {
+
+				$link_tec = '<a href="@todo" target="_blank">' . esc_html__( 'The Events Calendar', 'the-events-calendar' ) . '</a>';
+				$link_et = '<a href="@todo" target="_blank">' . esc_html__( 'Events Tickets', 'the-events-calendar' ) . '</a>';
+				$help->add_section_content( 'extra-help', sprintf( __( 'If you have tried the above steps and are still having trouble, you can post a new thread to our WordPress.org forums for %1$s or %2$s. Our support staff monitors these forums once a week and would be happy to assist you there. ', 'the-events-calendar' ), $link_tec, $link_et ), 20 );
+
+				$link = '<a href="@todo" target="_blank">' . esc_html__( 'premium support on our website', 'the-events-calendar' ) . '</a>';
+				$help->add_section_content( 'extra-help', sprintf( __( '<strong>Looking for more immediate support?</strong> We offer %s with the purchase of any of our premium plugins. Pick up a license and you can post there directly and expect a response within 24-48 hours during weekdays', 'the-events-calendar' ), $link ), 20 );
+
+			} elseif ( ! $help->is_active( 'events-calendar-pro' ) && ! $help->is_active( 'event-tickets' ) ) {
+
+				$link = '<a href="@todo" target="_blank">' . esc_html__( 'open-source forum on WordPress.org', 'the-events-calendar' ) . '</a>';
+				$help->add_section_content( 'extra-help', sprintf( __( 'If you have tried the above steps and are still having trouble, you can post a new thread to our %s. Our support staff monitors these forums once a week and would be happy to assist you there.', 'the-events-calendar' ), $link ), 20 );
+
+				$link_forum = '<a href="http://m.tri.be/4w/" target="_blank">' . esc_html__( 'premium support on our website', 'the-events-calendar' ) . '</a>';
+				$link_plus = '<a href="@todo" target="_blank">' . esc_html__( 'Events Calendar PRO', 'the-events-calendar' ) . '</a>';
+				$help->add_section_content( 'extra-help', sprintf( __( '<strong>Looking for more immediate support?</strong> We offer %1$s with the purchase of any of our premium plugins (like %2$s). Pick up a license and you can post there directly and expect a response within 24-48 hours during weekdays.', 'the-events-calendar' ), $link_forum, $link_plus ), 20 );
+
+			} else {
+
+				$link = '<a href="@todo" target="_blank">' . esc_html__( 'post a thread', 'the-events-calendar' ) . '</a>';
```

Comment:
Taking this and the other `@todo` hrefs to be intentional per [this comment](https://github.com/moderntribe/event-tickets/pull/53#discussion_r44847048).